scargo/docker_base/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+---
+version: '3.9'
+
+services:
+  scargo-x86:
```

Comment:
At the end this wont be true, each project because of the custom layer shall have is separate name as otherwise they will overwrite eachother